shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/metadata/schema/builder/loader/dialect/H2TableMetaDataLoader.java

```diff
@@ -164,14 +178,15 @@ private String getPrimaryKeyMetaDataSQL(final Collection<String> existedTables)
         return result;
     }
 
-    private String getGeneratedInfoSQL(final Collection<String> existedTables) {
-        return existedTables.isEmpty() ? GENERATED_INFO_SQL
-                : String.format(GENERATED_INFO_SQL_WITH_EXISTED_TABLES, existedTables.stream().map(each -> String.format("'%s'", each)).collect(Collectors.joining(",")));
+    private String getGeneratedInfoSQL(final Collection<String> tables, final boolean isExclude) {
```

Comment:
conditional-operator appear two times in one clause ? I believe this is easy to make people confused.